runtime.md

```diff
@@ -14,4 +14,40 @@ Runs a process in a container. Can be invoked several times.
 
 Not sure we need that from runc cli. Process is killed from the outside.
 
-This event needs to be captured by runc to run onstop event handlers.
+This event needs to be captured by oc to run onstop event handlers.
+
+## Hooks
+Hooks allow one to run code before/after various lifecycle events of the container.
+The state of the container is passed to the hooks over stdin, so the hooks could get the information they need to do their work.
```

Comment:
We will document that once we finalize the format which is being discussed in other issues.